{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--8fd16314-5993-4c8e-9dc8-c4a88ec265d0",
    "created": "2024-08-13T16:02:55.154996Z",
    "modified": "2024-08-13T16:02:55.155Z",
    "name": "No Title Available",
    "description": "When importing repos via URL, one time use git credentials were persisted beyond the expected time window in Gitaly 1.79.0 or above.",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2020-13353"
        }
    ],
    "credits": [
        {
            "lang": "en",
            "value": "This vulnerability has been discovered internally by the GitLab team"
        }
    ],
    "metrics": [
        {
            "cvssV3_1": {
                "attackComplexity": "HIGH",
                "attackVector": "LOCAL",
                "availabilityImpact": "NONE",
                "baseScore": 2.5,
                "baseSeverity": "LOW",
                "confidentialityImpact": "LOW",
                "integrityImpact": "NONE",
                "privilegesRequired": "HIGH",
                "scope": "CHANGED",
                "userInteraction": "NONE",
                "vectorString": "CVSS:3.1/AV:L/AC:H/PR:H/UI:N/S:C/C:L/I:N/A:N",
                "version": "3.1"
            }
        }
    ],
    "references": [
        {
            "tags": [
                "x_refsource_MISC"
            ],
            "url": "https://gitlab.com/gitlab-org/gitaly/-/issues/2882"
        },
        {
            "tags": [
                "x_refsource_CONFIRM"
            ],
            "url": "https://gitlab.com/gitlab-org/cves/-/blob/master/2020/CVE-2020-13353.json"
        }
    ]
}